HaskellDates.hsproj/HaskellDates.hs

```diff
@@ -0,0 +1,62 @@
+import Data.Time
+import Data.Time.Calendar.OrdinalDate
+import Control.Applicative
+
+data Event = Event {
+  date :: Day,
+  daysOfWeek :: [Int],
+  week :: Int
+}
+
+-- WeekDay for debugging purposes
+newtype WeekDay = WeekDay Int
```

Comment:
I was thinking about doing that but this type is just for debugging so I can see a printed word rather than a number. I'll probably ditch it when I actually use this code in my side project.